components/eamxx/scripts/atmchange

```diff
@@ -13,119 +13,11 @@ sys.path.append(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__
 sys.path.append(os.path.dirname(os.path.realpath(__file__)))
 
 from eamxx_buildnml_impl import check_value, is_array_type
-from atm_manip import expect, get_xml_node
+from atm_manip import expect, get_xml_node, atm_config_chg_impl
```

Comment:
I guess importing `expect` in `atm_manip` makes it available here, but maybe we can move the import of `expect` to the next line, for clarity. But not really important.